contrib/operator-tools/linux-tor-prio.sh

```diff
@@ -7,16 +7,16 @@
 ############################### README #################################
 
 # This script provides prioritization of Tor traffic below other
-# traffic on a Linux server. It has two modes of operation: UID based 
```

Comment:
Please make the formatting changes in a separate commit.

Reply:
Fixed.